### ISSUE TITLE
Can't join discord community

### ISSUE THREAD

**Issue by SantiagoIvan** [ayush714/ML001-Project-Sources-Code-and-Learning-Materials#4]
Hi !  New learner here :)
I was trying to join to the discord community, but the invitation link has expired :(

**Comment by BoseSj** [comment]
Yeah, the discord link in [README](https://github.com/ayush714/ML001-Project-Sources-Code-and-Learning-Materials#readme) seems to be broken, can't join using it. Please fix it. @ayush714 
Also to be mentioned here, the provided link for Instagram and Twitter is also invalid.

**Comment by Vaelico333** [comment]
I'd like to join the discord community too, please fix it!
